contrib/executor/tracetest/tracetest-test.yaml

```diff
@@ -0,0 +1,19 @@
+type: Test
+spec:
+  name: Pokeshop - List
+  description: Get a Pokemon
+  trigger:
+    type: http
+    httpRequest:
+      url: http://demo-pokemon-api.demo/pokemon?take=20&skip=0
+      method: GET
+      headers:
+        - key: Content-Type
+          value: application/json
+  # specs:
```

Comment:
should it be removed?

Reply:
the whole file can be found in the docs, will remove it for now